vite.config.js

```diff
@@ -1,17 +1,40 @@
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 import visualizer from 'rollup-plugin-visualizer';
+import { resolve } from "path";
+import { nodeResolve } from '@rollup/plugin-node-resolve'; 
+
+// Workaround for this open solid-client-authn bug:
+// https://github.com/inrupt/solid-client-authn-js/issues/3001
+// Once that issue is fixed and the solid-client-authn-js version is bumped
+// We can delete this function and all related code
+function withCustomResolver(
+  { find, exportConditions, mainFields }
+) {
+  const customResolver = nodeResolve({exportConditions, mainFields })
+  return { find, replacement: find, customResolver }
+}
 
 export default defineConfig({
-  base: '/PASS/',
   plugins: [
     react(),
     visualizer({
-      filename: 'bundleStats.html',
-      open: true
+      filename: 'bundleStats.html'
     })
   ],
+  resolve:{
+    alias:[
```

Comment:
I do like we're going to swap over to using alias for some of our imports. Helps cuts down the lines of text we might get from typing the relative pathing to certain files/directories.

Reply:
Yeah, and if we ever need more, we can just add them to the lists here and in eslint.rc.

Reply:
Haha, yeah 😆 

I'm good with this.